django/ReleaseAndStage/metadata.yml

```diff
@@ -1,5 +1,10 @@
+suggested: true
+tech-preview: false
 environments:
   - Stage
 stages:
-  - 'Build Release'
-  - 'Rollout to Stage'
+  - name: Build Release
+    description: creates a new version then builds and deploys the project into the maven repository 
+  - name : Rollout to Stage
```

Comment:
nitpick. inconsistent spacing. I see that some parts of the yaml  follows the usual convention of `key: value` while other have a space between `key` and `:`. Lets be consistent use `key: value` every where instead of `key : value`